CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing To SwaySwap
+
+Thanks for your interest in contributing to SwaySwap! This document outlines the process for installing dependencies and setting up SwaySwap for development, as well as some conventions on contributing to SwaySwap.
+
+If you run into any difficulties getting started, you can always ask questions on our [Discord](https://discord.gg/xfpK4Pe).
+
+## Setting up a development workspace
+
+See the [introduction](../introduction/index.md) section for instructions on installing and setting up the Sway toolchain. (This step is only necessary if contributing to smart contract development with Sway.)
+
+You will need to have `yarn`, or a similar package manager installed. For the purposes of this contributing, we will recommend what was used in the original development of the project.
+
+```sh
+yarn install
+```
+
+## Getting the repository
+
+1. Visit the [SwaySwap](https://github.com/FuelLabs/swayswap) repo and fork the project.
+2. Then clone your forked copy to your local machine and get to work.
+
+```sh
+git clone https://github.com/FuelLabs/swayswap
+cd swayswap
+```
+
+## Building and testing
+
+The following steps are for running and viewing the SwaySwap dapp in your preferred browser.
+
+Open the client folder and deploy the SwaySwap dapp for development mode:
+
+```sh
+cd client
+yarn start
+```
+
+To view SwaySwap in the browser, open [http://localhost:3000](http://localhost:3000).
+
+Congratulations! You've now got everything setup and are ready to start making contributions.
+
```

Comment:
I thin wee need to add the steps to;

- Run `docker compose up`
- Build `contracts` using `yarn build-contracts`
- Deploy `contracts` using `yarn deploy-contracts`
- Replace the `.env` config withe the correct `contract ids`.

Reply:
I think this got added to the readme recently